src/main/java/components/PanelHeader.java

```diff
@@ -0,0 +1,102 @@
+package components;
+
+import enums.Attribute;
+import enums.CssProperty;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import pages.CreateAccountPage;
+import pages.LoginPage;
+
+
+import java.util.List;
+
+import static utils.ColorUtils.getColorAsHex;
+
+public class PanelHeader {
+    private final WebDriver driver;
+
+    public PanelHeader(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    private final By createAccountHeaderLink = By.cssSelector(".panel.header .header.links li:last-child");
+    private final By signInHeaderLink = By.cssSelector(".header.links .authorization-link");
+    private final By thisIsDemoSiteSection = By.cssSelector(".message.global.demo");
```

Comment:
thisIsDemoSiteSection does not descriptive name of component. Should be more like `topDemoInfoBanner`